vars/githubPublishRelease.groovy

```diff
@@ -86,7 +90,18 @@ void call(Map parameters = [:]) {
         new Utils().pushToSWA([step: STEP_NAME], config)
 
         withCredentials([string(credentialsId: config.githubTokenCredentialsId, variable: 'TOKEN')]) {
-            def releaseBody = config.releaseBodyHeader?"${config.releaseBodyHeader}<br />":''
+
+            def releaseBodyHeader = ''
+            if (config.releaseBodyHeader) {
+                releaseBodyHeader = GStringTemplateEngine.newInstance()
```

Comment:
Wondering if this is cps safe. Any considerations in that regard?

Reply:
GStringTemplateEngine is Serializable hence should not have an impact on CPS.